src/main/webapp/app/exercises/shared/plagiarism/plagiarism-inspector/plagiarism-inspector.component.html

```diff
@@ -57,7 +47,16 @@ <h4 class="plagiarism-page-title" (click)="showOptions = !showOptions" [class.ac
                     <div class="plagiarism-option-label" jhiTranslate="artemisApp.plagiarism.minimum-score">Minimum Score</div>
                 </label>
                 <fa-icon [icon]="'question-circle'" placement="bottom" [ngbTooltip]="'artemisApp.plagiarism.minimum-score-tooltip' | artemisTranslate" container="body"></fa-icon>
-                <input required type="number" [disabled]="!enableMinimumScore" class="form-control" min="0" max="100" id="plagiarism-minimum-score" [(ngModel)]="minimumScore" />
+                <input
+                    required
+                    type="number"
+                    [disabled]="!enableMinimumScore"
+                    class="form-control pr-0"
```

Comment:
Note: `pr-0` will need to be renamed if we merge this after #3470. If this is merged first #3470 needs to be adjusted.

Reply:
Thanks for the hint, I'll change it